tests/Domain/data/provider_valid.php

```diff
@@ -0,0 +1,6 @@
+<?php declare(strict_types = 1);
+
+return [
+    'name' => 'Provider Name Test',
+    'tlds' => ['.com', '.net'],
```

Comment:
Please change this to `['com', 'net']`

Reply:
Done: https://github.com/sandwave-io/realtimeregister-php/pull/22/commits/9743603880caf5d6044ac7abc219848654014498#diff-d26e3d12584eb2d647e97650dde7b1a4